flowkit/flowkit.go

```diff
@@ -871,6 +871,17 @@ func (f *Flowkit) GetTransactionsByBlockID(
 
 	txRes, err := f.gateway.GetTransactionResultsByBlockID(blockID)
 	if err != nil {
```

Comment:
I would suggest inverting the condition to make the code easier to read.

if no error return else try fetching one by one.

Reply:
Totally agree.